cli/command/container/rm.go

```diff
@@ -45,13 +45,22 @@ func runRm(dockerCli *command.DockerCli, opts *rmOptions) error {
 	ctx := context.Background()
 
 	var errs []string
-	for _, name := range opts.containers {
-		if name == "" {
+	options := types.ContainerRemoveOptions{
+		RemoveVolumes: opts.rmVolumes,
+		RemoveLinks:   opts.rmLink,
+		Force:         opts.force,
+	}
+
+	errChan := parallelOperation(ctx, opts.containers, func(ctx context.Context, id string) error {
+		if id == "" {
 			return fmt.Errorf("Container name cannot be empty")
```

Comment:
Not indeed. It can be either container id or container name, up to input from user. :smile: